evm_arithmetization/src/generation/mod.rs

```diff
@@ -280,6 +292,12 @@ pub fn generate_traces<F: RichField + Extendable<D>, const D: usize>(
     let public_values = PublicValues {
         trie_roots_before,
         trie_roots_after,
+        #[cfg(feature = "cdk_erigon")]
+        burn_addr: if let Some(burn_addr) = inputs.burn_addr {
+            U256::from_big_endian(&burn_addr.0)
+        } else {
+            U256::MAX
+        },
```

Comment:
storing it as a `U256` is a bit confusing, given it's an `Address`, and yields additional limbs that add overhead (challenger / wire connections, ...)

Reply:
The problem is that the burn address can be anything, so I can't set the `None` case to be H256::MAX for example. So I use U256::MAX to signal that there is no burn address

Reply:
When you said you can't set it to H256::MAX you meant H160 no?